tests/integration/pilot/ingress_test.go

```diff
@@ -637,79 +638,6 @@ spec:
 				})
 			})
 
-			// TODO we could add istioctl as well, but the framework adds a bunch of stuff beyond just `istioctl install`
-			// that mess with certs, multicluster, etc
-			t.NewSubTest("helm").Run(func(t framework.TestContext) {
-				gatewayNs := namespace.NewOrFail(t, namespace.Config{Prefix: "custom-gateway-helm", Inject: inject})
```

Comment:
Why are we removing a test?

Reply:
I removed it based on the review comments - https://github.com/istio/istio/pull/52974#pullrequestreview-2277956693